pom.xml

```diff
@@ -443,7 +443,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-source-plugin</artifactId>
-                    <version>3.2.1</version>
+                    <version>3.1.0</version>
```

Comment:
Why was this downgraded?

Reply:
Because I'm grasping at straws trying to figure out why a simple dependency won't resolve.